pkgs/development/libraries/cubeb/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib, stdenv, fetchFromGitHub
+, cmake
+, gtest
+, pkg-config
+, jack2
+, pulseaudio
+, sndio
+, speexdsp
+, lazyLoad ? true
+}:
+
+let
+  version = "unstable-2022-10-18";
+  rev = "27d2a102b0b75d9e49d43bc1ea516233fb87d778";
+  hash = "sha256-q+uz1dGU4LdlPogL1nwCR/KuOX4Oy3HhMdA6aJylBRk=";
```

Comment:
Any reason for not defining those inside the `mkDerivation` attrset instead?

Reply:
Thought that looks neater when you have to change 3 fields when updating. If you don't like it we can move it in.